src/lpython/semantics/python_ast_to_asr.h

```diff
@@ -8,7 +8,7 @@ namespace LCompilers::LPython {
 
     Result<ASR::TranslationUnit_t*> python_ast_to_asr(Allocator &al, LocationManager &lm, SymbolTable* symtab,
         LPython::AST::ast_t &ast, diag::Diagnostics &diagnostics, CompilerOptions &compiler_options,
-            bool main_module, std::string module_name, std::string file_path, bool allow_implicit_casting=false);
+            bool main_module, std::string module_name, std::string file_path, bool allow_implicit_casting=false, bool is_interactive=false);
```

Comment:
You can also pass the `eval_count` here and have it as a class member. Then for `body visitor` inside `visit_Module` (assuming we are reusing this instead of defining `visit_Interactive`), you can use the `eval_count` for naming `global_init` and `global_stmts` if `interactive = true`. (For example simply append the eval_count to the end if interactive is true).